examples/models/llama/attention.py

```diff
@@ -243,14 +244,18 @@ def forward(
         k = k.view(bsz, seqlen, self.n_local_kv_heads, self.head_dim)
         v = v.view(bsz, seqlen, self.n_local_kv_heads, self.head_dim)
 
+        if self.use_qk_norm and self.qk_norm_before_rope:
```

Comment:
might've been cleaner to make it `qk_norm_mode` that can be one of None, BEFORE_ROPE, AFTER_ROPE. don't know if we have backward compatibility constraints preventing that

Reply:
Yea i agree, this would be cleaner. @jackzhxng let's follow up with this change.